deploy-olm-operator.yml

```diff
@@ -2,7 +2,7 @@
 - name: Deploy the operator on an OCP cluster with OLM as part of the operator testing pipeline
   hosts: all
   become: false
-  gather_facts: false
+  gather_facts: true
```

Comment:
We don't need to enable `gather_facts` since this is being run explicitly on localhost and we're not using any ansible* facts (or I think you were but since removed them).

Reply:
Yes, thank you